src/components/ImportFile/ImportFileContent/ImportFileReject.js

```diff
@@ -0,0 +1,55 @@
+import ClassNames from 'classnames';
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Icon } from '../../Icon';
+import { Button } from '../../Button';
+
+const ImportFileReject = ({
+  rejectedFiles,
+  reset,
+  text,
+  icon,
+  className,
+  ...props
```

Comment:
no need for this spread.

Reply:
by no need you mean not give the consumer a choice for text/icon?